### PR TITLE
display ssh key details, if available

### PR DESCRIPTION
**Pull request by herb**
this displays ssh key details to the user. if available. this is part 1 of 2 changes to provide the ability to audit all user ssh keys.

### CODE REVIEW

File: grouper/fe/util.py [hunk 1]
```diff
@@ -276,46 +277,61 @@ def print_date(date_obj):
...
-    ("day", 86400),
-    ("hour", 3600),
-    ("minute", 60),
-    ("second", 1),
-]
+_DELTA_COMPONENTS = ["year", "month", "day", "hour", "minute", "second"]
 
-
-def delta_str(date_obj):
+def expires_when_str(date_obj, utcnow_fn=datetime.utcnow):
```
Comment (lfaraone) [inline grouper/fe/util.py]: Is this change related to the overall `ssh` key details?

**Comment by herb** [comment]
splitting this guy into two commits. first of which: https://github.com/dropbox/grouper/pull/73
